ui/packages/catalog-ui-search/src/main/webapp/component/result-item/result-item.less

```diff
@@ -59,6 +60,7 @@
     width: ~'calc(100% - 0.5*@{minimumSpacing})';
     position: relative;
     left: 0.5 * @minimumSpacing;
+    background: fade(#ffffff, 7%);
```

Comment:
After updating the code, we're only using this in two places, and it's a special place. If we made it generic, we'd want to be able to change the color and the percentage, and at that point, I'm not sure it would be helpful?